include/jwt-cpp/jwt.h

```diff
@@ -2606,12 +2607,166 @@ namespace jwt {
 #endif
 	};
 
+	namespace verify_ops {
+		template<typename json_traits>
+		struct verify_context {
+			verify_context(date ctime, const decoded_jwt<json_traits>& j, size_t l)
+				: current_time(ctime), jwt(j), default_leeway(l) {}
+			// Current time, retrieved from the verifiers clock and cached for performance and consistency
+			date current_time;
+			// The jwt passed to the verifier
+			const decoded_jwt<json_traits>& jwt;
+			// The configured default leeway for this verification
+			size_t default_leeway{0};
+
+			// The claim key to apply this comparision on
+			typename json_traits::string_type claim_key{};
+		};
+
+		/**
+		 * This is the default operation and does case sensitive matching
+		 */
+		template<typename json_traits>
+		struct equals_claim {
+			basic_claim<json_traits> expected;
+			void operator()(const verify_context<json_traits>& ctx, std::error_code& ec) {
```

Comment:
I think we can add a lot of `const` in this section

```suggestion
			const basic_claim<json_traits> expected;
			void operator() const (const verify_context<json_traits>& ctx, std::error_code& ec) {
```